### ISSUE TITLE
username and password not working on AfreecaTV

### ISSUE THREAD

**Issue by eviljust** [yt-dlp/yt-dlp#10438]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Singapore

### Provide a description that is worded well enough to be understood

yt-dlp.exe --username "USERNAME" --password "PASSWORD" --get-url https://vod.afreecatv.com/player/

Error message I got is "WARNING: [afreecatv] In accordance with local laws and regulations, underage users are restricted from watching adult content. Only content suitable for all ages will be downloaded. Use --username and --password, --netrc-cmd, or --netrc (afreecatv) to provide account credentials"

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
C:\Users\justinho\Desktop>yt-dlp.exe -vU --username username --password password --get-url https://vod.afreecatv.com/player/
[debug] Command-line config: ['-vU', '--username', 'PRIVATE', '--password', 'PRIVATE', '--get-url', 'https://vod.afreecatv.com/player/129944563']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.07.09 from yt-dlp/yt-dlp [7ead7332a] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 7.0-full_build-www.gyan.dev (setts), ffprobe 7.0-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.07.04, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.35.5, urllib3-2.2.2, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1834 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: stable@2024.07.09 from yt-dlp/yt-dlp
yt-dlp is up to date (stable@2024.07.09 from yt-dlp/yt-dlp)
[afreecatv] Logging in
[afreecatv] Extracting URL: https://vod.afreecatv.com/player/
[afreecatv] 129944563: Downloading JSON metadata
[afreecatv] 129944563: Downloading part 1 m3u8 information
[afreecatv] 129944563: Downloading part 2 m3u8 information
[afreecatv] 129944563: Downloading part 3 m3u8 information
[afreecatv] 129944563: Downloading part 4 m3u8 information
WARNING: [afreecatv] In accordance with local laws and regulations, underage users are restricted from watching adult content. Only content suitable for all ages will be downloaded. Use --username and --password, --netrc-cmd, or --netrc (afreecatv) to provide account credentials
```

**Comment by bashonly** [comment]
Please don't edit your log / redact the URL.

yt-dlp is downloading 4 parts of the VOD in your log. How many parts does this VOD have in your browser when logged in? 

Also, could you test this completely restricted VOD with yt-dlp login:
```
yt-dlp --username "YOURUSERNAME" --password "YOURPASSWORD" -vF "https://vod.afreecatv.com/player/70395877"
```

**Comment by eviljust** [comment]
I found one age restricted VOD which only have one part to make it easier for me to try.


C:\Users\xxxxxx\Desktop>yt-dlp.exe --username "SGP0524" --password "apl664fe2f" -vF "https://vod.afreecatv.com/player/128831147"
[debug] Command-line config: ['--username', 'PRIVATE', '--password', 'PRIVATE', '-vF', 'https://vod.afreecatv.com/player/128831147']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2024.07.09 from yt-dlp/yt-dlp [7ead7332a] (win_exe)
[debug] Python 3.8.10 (CPython AMD64 64bit) - Windows-10-10.0.19045-SP0 (OpenSSL 1.1.1k  25 Mar 2021)
[debug] exe versions: ffmpeg 7.0-full_build-www.gyan.dev (setts), ffprobe 7.0-full_build-www.gyan.dev
[debug] Optional libraries: Cryptodome-3.20.0, brotli-1.1.0, certifi-2024.07.04, curl_cffi-0.5.10, mutagen-1.47.0, requests-2.32.3, sqlite3-3.35.5, urllib3-2.2.2, websockets-12.0
[debug] Proxy map: {}
[debug] Request Handlers: urllib, requests, websockets, curl_cffi
[debug] Loaded 1834 extractors
[afreecatv] Logging in
[afreecatv] Extracting URL: https://vod.afreecatv.com/player/128831147
[afreecatv] 128831147: Downloading JSON metadata
ERROR: [afreecatv] 128831147: Only users older than 19 are able to watch this video. Use --username and --password, --netrc-cmd, or --netrc (afreecatv) to provide account credentials
  File "yt_dlp\extractor\common.py", line 740, in extract
  File "yt_dlp\extractor\afreecatv.py", line 235, in _real_extract
  File "yt_dlp\extractor\common.py", line 1245, in raise_login_required

**Comment by bashonly** [comment]
Will be fixed by #10440. See the PR description for an explanation of what was going wrong.

@eviljust FYI the original version of your last comment contained your username and password. I have edited the comment and removed them, and I've deleted the original version of the comment. However, the original version of the comment was emailed to anyone subscribed to this issue or to the yt-dlp repository. I'd recommend changing your password ASAP
